code/modules/mob/mob.dm

```diff
@@ -1390,3 +1390,29 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 ///Force set the mob nutrition
 /mob/proc/set_nutrition(change)
 	nutrition = max(0, change)
+
+/mob/clean_blood(clean_hands = TRUE, clean_mask = TRUE, clean_feet = TRUE)
+	. = ..()
+	if(bloody_hands && clean_hands)
+		bloody_hands = 0
```

Comment:
FALSE here

Reply:
Not a bool

Reply:
TIL this game has the worst variable naming conventions